lib/routes/routeBackbeat.js

```diff
@@ -706,16 +706,6 @@ function routeBackbeat(clientIP, request, response, log) {
                 return backbeatRoutes[request.method][request.resourceType]
                     [request.query.operation](request, response, log, next);
             }
-            const versioningConfig = bucketInfo.getVersioningConfiguration();
```

Comment:
Allow the user to be responsible for setting versioning on the destination bucket, since at this point we don't know whether the source is an NFS bucket.